### PR TITLE
Translate user manual, table-structure for event-trace.

### PR DESCRIPTION
**Pull request by dongzl**
For #1138 .

### CODE REVIEW

File: docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.cn.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ JOB_EXECUTION_LOG 记录每次作业的执行历史。
 分为两个步骤：
 
 1. 作业开始执行时向数据库插入数据，除 failure_cause 和 complete_time 外的其他字段均不为空。
-1. 作业完成执行时向数据库更新数据，更新 is_success, complete_time 和 failure_cause(如果作业执行失败)。
+2. 作业完成执行时向数据库更新数据，更新 is_success, complete_time 和 failure_cause(如果作业执行失败)。
```
Comment (terrymanu) [inline docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.cn.md]: Please do not modify here, it is the correct md syntax.
The number will auto increment

File: docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.en.md [hunk 2]
```diff
@@ -4,4 +4,45 @@ weight = 4
...
+| failure_cause    | VARCHAR(2000) | No        | The reason for execution failure                                           |
+| is_success       | BIT           | Yes       | Execute successfully or not                                           |
+| start_time       | TIMESTAMP     | Yes       | Job start time                                        |
+| complete_time    | TIMESTAMP     | No        | Job end time                                        |
+
+`JOB_EXECUTION_LOG` records the execution history of each job.
+There are two steps:
+
+1. When the job is executed, program will create one record in the `JOB_EXECUTION_LOG`, and all fields except `failure_cause` and `complete_time` are not empty.
+2. When the job completes execution, program will update the record, update the columns of `is_success`, `complete_time` and `failure_cause`(if the job execution fails).
```
Comment (terrymanu) [inline docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.en.md]: Please leave here as `1.`

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.cn.md (hunk 1 above)
Comment (dongzl) [reply docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.cn.md]: OK, fix.

File: docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.en.md (hunk 2 above)
Comment (dongzl) [reply docs/content/user-manual/elasticjob-lite/usage/event-trace/table-structure.en.md]: OK, fix.

**Review by terrymanu — APPROVED** [review]

**Pull request merged**